### PR TITLE
Updated Acceptance Tests

### PR DESCRIPTION
**Pull request by clintonb**
- Authenticating via OAuth2 for E-Commerce API
- Using patched bok-choy dependency to fix validation of URLs with basic auth credentials
- Fixed bug that always attempted to create an LMS auto-auth user, even if credentials supplied on command line

@jimabramson @rlucioni

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -15,4 +15,4 @@ logutils==0.3.3
 pycountry==1.10
 requests==2.6.0
 
-git+https://github.com/edx/ecommerce-api-client.git@0.1.0#egg=ecommerce-api-client
+git+https://github.com/edx/ecommerce-api-client.git@oauth2#egg=ecommerce-api-client
```
Comment (rlucioni) [inline requirements/base.txt]: Do you need to include a version number on the egg here, as in [#7932](https://github.com/edx/edx-platform/pull/7932/files#diff-9280d61660044201fa35456d5ae4ee74R51)?

Comment (clintonb) [inline requirements/base.txt]: Updated to use the release, now that it has been merged.

**Comment by rlucioni** [comment]
:+1:

File: requirements/test.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Packages required for testing
 -r base.txt
 
-bok-choy==0.3.3
+git+https://github.com/edx/bok-choy.git@e4e5934b9645e389d14ef22587e8462be4c1cd74#egg=bok_choy
+#bok-choy==0.3.3
```
Comment (jimabramson) [inline requirements/test.txt:5]: remove this line?

Comment (clintonb) [inline requirements/test.txt:5]: This line will be restored when a new version of bok-choy is released.

**Pull request merged**